curriculum/challenges/espanol/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-operator.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 56533eb9ac21ba0edf2244d6
-title: Comparison with the Less Than Operator
+title: Comparación con el operador "menor que"
```

Comment:
@RafaelDavisH FYI - Should `menor que` have double quotes around since there are not double quotes in the English title?